core/sts-3.3.md

```diff
@@ -0,0 +1,129 @@
+---
+title: IRCv3.3 Strict Transport Security
+layout: spec
+copyrights:
+  -
+    name: "Attila Molnar"
+    period: "2015"
+    email: "attilamolnar@hush.com"
+---
+
+## Description
+
+Strict Transport Security (STS) is a mechanism which allows servers to
+communicate that complying clients should only connect to them over a secure
+connection.
+
+The policy is communicated to the client via the `sts` capability and should
+be processed by the client at capability negotiation time.
+
+The value of the capability specifies the duration during which the client
+MUST only connect to the server securely (using TLS, aka. SSL), and the port
+number to use for it.
+
+Secure connections must use a TLS server certificate that is signed with a
+certificate from a reputable CA chain, using either the OS trust root or a
+list such as Mozilla's.
```

Comment:
Yes, this is why I initially tried to avoid dealing with what should be a "secure" connection in this spec, but realized that it's a better idea to talk about it than to let client devs think it's fine to auto-accept a self-signed cert.